pkg/config/orderer.go

```diff
@@ -36,6 +36,11 @@ type Orderer struct {
 	// Capabilities is a map of the capabilities the orderer supports.
 	Capabilities map[string]bool
 	Policies     map[string]Policy
+	// State defines the orderer mode of operation.
+	// Normal is during normal operation
+	// Maintenance is during consensus type migration
+	// Options: `STATE_NORMAL` and `STATE_MAINTENANCE`
```

Comment:
Do these strings warrant constants / typed values?

Reply:
It does, I'll update in a follow on commit.